README.md

```diff
@@ -1 +1,68 @@
 # containerlab-vms
+
+This project is meant to help building containerlab compatible containers, running virtual routers inside of them.  This has been highly inspired by the vrntelab project.  The working principle is the same as the aforementioned project, but the inner structure of the code has been reviewed.
```

Comment:
I think you mean revised here? I would also add something like:

... has been revised to fix stability issues we encountered.

Reply:
Done in https://github.com/inmanta/containerlab-vms/pull/1/commits/51586612f99d651afe2c4280d0af16574c66739d